src/main/resources/db/changelog/v.0.1/db.changelog-users.yaml

```diff
@@ -0,0 +1,64 @@
+databaseChangeLog:
+  - changeSet:
+      id: 10-add-user-role-enum
+      author: elizaveta_dubogryz
+      dbms: postgresql
+      sql: CREATE TYPE USER_ROLE as ENUM ('USER', 'ADMIN');
+  - changeSet:
+      id: 11-create-users-table
+      author: elizaveta_dubogryz
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: login
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: password
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: first_name
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: last_name
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: email
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: telephone
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: role
+                  type: USER_ROLE
+                  constraints:
+                    nullable: false
+              - column:
+                  name: country_id
```

Comment:
Why do you need country here if you have city. City should have reference to country

Reply:
+